Formula/sigrok-cli.rb

```diff
@@ -0,0 +1,38 @@
+class SigrokCli < Formula
+  desc "Sigrok command-line interface to use logic analyzers and more"
+  homepage "https://sigrok.org/"
+  url "https://sigrok.org/download/source/sigrok-cli/sigrok-cli-0.7.2.tar.gz"
+  sha256 "71d0443f36897bf565732dec206830dbea0f2789b6601cf10536b286d1140ab8"
+  license "GPL-3.0-or-later"
+
+  livecheck do
+    url "https://sigrok.org/download/source/sigrok-cli/"
+    regex(/sigrok-cli-(\d+(?:\.\d+)+)\.t/i)
```

Comment:
Would be better to use href attribute for the livecheck (common pattern that we use)
```suggestion
    regex(/href=.*?sigrok-cli-(\d+(?:\.\d+)+)\.t/i)
```

cc @samford